src/annotator/hypothesis-injector.js

```diff
@@ -100,7 +114,7 @@ export async function injectClient(frame, config, frameId) {
   };
 
   const configElement = document.createElement('script');
-  configElement.className = 'js-hypothesis-config';
+  configElement.className = 'js-hypothesis-config js-temp-config';
```

Comment:
I wonder if there's a different class name we could use instead of `js-temp-config`. This is mildly misleading (I think?) because this script node will not go away in some cases (e.g. it sticks around in the VitalSource EPUB example page if you don't unload the client). It feels a little funky to have a "temp" script which might live for an indefinite amount of time...

Reply:
The important distinction here really is between configuration provided by the page author, which should not be removed when the client unloads, vs config added by the `injectClient` function, which should. Another way of handling this would have to have a `data-remove-on-unload` attribute to mark configuration objects that should be removed when the client unloads. Thoughts?

Reply:
Yep! I've found that I'm doing transition from JS classes to data attributes in other places lately, so this seems consistent. Your suggested attribute name seems fine here!

Reply:
OK, I have replaced the `js-temp-config` class with a `data-remove-on-unload` attribute.